website/app/templates/engineering.hbs

```diff
@@ -273,46 +288,126 @@
   </p>
+  <p>Since this is just an SVG asset, there are no
+    <em>props</em>
+    that can be passed to it. You should refer to the
+    <a
+      href="https://react-components.vercel.app/components/inlinesvg"
+      target="_blank"
+      rel="noopener noreferrer"
+    >&lt;InlineSvg&gt;</a>
+    documentation to know how to apply color and size to the SVG icon.
+  </p>
+
+  <h3 class="ds-h3" id="react-svg-component"><a href="#react-svg-component" class="ds-a" rel="external">&sect;</a>
+    React/SVG</h3>
+
   <p>
-    Authors should also follow the following guidelines:
-    <ul class="ds-ul">
-      <li class="ds-li">The icons are sized as 16x16(px) and 24x24(px) and should not be used at different sizes without
-        a design consult.</li>
-      <li class="ds-li">The icons do not have a unique id generated; authors should take precautions to avoid
-        <a
-          href="https://www.w3.org/TR/WCAG20-TECHS/F77.html"
-          class="ds-a"
-          target="_blank"
-          rel="noopener noreferrer"
-        >related accessibility conformance failures</a>.</li>
-    </ul>
+    Single icons can be imported and used directly as SVG files using the
+    <a
+      href="https://react-components.vercel.app/components/inlinesvg"
+      target="_blank"
+      rel="noopener noreferrer"
+    >&lt;InlineSvg&gt;</a>
+    provided by the
+    <a
+      href="https://github.com/hashicorp/react-components"
+      target="_blank"
+      rel="noopener noreferrer"
+    >@hashicorp/react-components</a>
+    library:
+    {{! template-lint-disable 'no-potential-path-strings' }}
+    {{! prettier-ignore-start }}
+    <CodeBlock
+      @language="js"
+      @code="
+          // import the React file (TypeScript)
+          const FlightArrowRight = require('@hashicorp/flight-icons/svg-react/arrow-right-16');
+
+          // elsewhere in the file
+          <FlightArrowRight />
+        "
+    />
+    {{! prettier-ignore-end }}
+    {{! template-lint-enable 'no-potential-path-strings' }}
+    <em class="ds-em">Notice: the code above is an example, please update it accordingly to your codebase.</em>
+  </p>
+  <h3 class="ds-h3">Props</h3>
+  <p>The component exposes the following
+    <em>props</em>:
+    <ol>
+      <li><code>color</code>
+        - the color (applied as
+        <em>fill</em>) to the SVG - by default is `currentColor` but any valid HTML/CSS color is accepted</li>
+      <li><code>title</code>
+        - the title of the SVG - by default the icon has an
+        <em>aria-hidden</em>
+        attribute applied to it, because is expected to be used
+        <em>in contex</em>
+        (see
+        <a href="#accessibility">§ Accessibility</a>); if instead you need to use it without text associated to it, you
+        have to pass a
+        <em>title</em>
+        attribute to make it accessible.</li>
+      <li><code>...props</code> - any other <em>prop</em> passed to the component will be applied via spread</li>
```

Comment:
```suggestion
      <li><code class="ds-code">color</code>
        - the color (applied as
        <em>fill</em>) to the SVG - by default is `currentColor` but any valid HTML/CSS color is accepted</li>
      <li><code class="ds-code">title</code>
        - the title of the SVG - by default the icon has an
        <em>aria-hidden</em>
        attribute applied to it, because is expected to be used
        <em>in contex</em>
        (see
        <a href="#accessibility">§ Accessibility</a>); if instead you need to use it without text associated to it, you
        have to pass a
        <em>title</em>
        attribute to make it accessible.</li>
      <li><code class="ds-code">...props</code> - any other <em>prop</em> passed to the component will be applied via spread</li>
```

non-blocking suggestion add same styling

Reply:
Oh, my bad!! Didn't notice (remember) we have a special class for it.